docs/machine-learning/tutorials/sentiment-analysis.md

```diff
@@ -113,29 +134,41 @@ Remove the existing class definition and add the following code, which has two c
 
 `SentimentData` is the input dataset class and has a `float` (`Sentiment`) that has a value for sentiment of either positive or negative, and a string for the comment (`SentimentText`). Both fields have `Column` attributes attached to them. This attribute describes the order of each field in the data file, and which is the `Label` field. `SentimentPrediction` is the class used for prediction after the model has been trained. It has a single boolean (`Sentiment`) and a `PredictedLabel` `ColumnName` attribute. The `Label` is used to create and train the model, and it's also used with a second dataset to evaluate the model. The `PredictedLabel` is used during prediction and evaluation. For evaluation, an input with training data, the predicted values, and the model are used.
 
-In the *Program.cs* file, replace the `Console.WriteLine("Hello World!")` line with the following code in the `Main` method:
+In the *Program.cs* file, change the `Main` method signature by replacing `void` with `async Task`, as in the following example:
+
+```csharp
```

Comment:
you might want to pull this from the snippet file. Do you see how the rest of the topic lost its formatting here? (e.g. async on line 143 is not blue, etc.)

Reply:
If I do, then they have to enter all the code for the next several steps then as well.

Reply:
I put the braces in and the formatting returned. Hurrah!